client/blocks/daily-post-button/test/index.jsx

```diff
@@ -32,11 +38,11 @@ describe( 'DailyPostButton', () => {
 					markPostSeen={ markPostSeen }
 				/>
 			);
-			expect( dailyPostPrompt.type() ).toBeNull();
+			expect( container.firstChild ).not.toBeInTheDocument();
 		} );
 
 		test( 'renders as a span tag by default', () => {
```

Comment:
Not sure about this one. User's certainly don't care whether it's a `<span />` but it's dependent on `tagName` which should be relevant in cases where the prop is set. Shall we keep that test or is it enough to rely on the test after this one which sets `tagName` explicitly? /cc @tyxla